Google.PowerShell/Storage/GcsObject.cs

```diff
@@ -289,39 +289,39 @@ private void UploadDirectoryToGCSBucket(string directory, Dictionary<string, str
         {
```

Comment:
The .NET style guidelines dictate that it should be `Gcs` not `GCS`. Given that the context is pretty clear, why not just call this `UploadDirectory` and leave it at that? (Rename other methods if they do not obey the all-mighty style guide.)